AmethystAPI/src/amethyst/Utility.h

```diff
@@ -1,7 +1,8 @@
 #pragma once
 #include <string>
 
-/**
- * Returns the path to the amethyst UWP folder
-*/
-std::string GetAmethystFolder();
\ No newline at end of file
+/// Returns the path to the amethyst UWP folder
+std::string GetAmethystFolder();
```

Comment:
Why change the /** */ comment, im just curious? I've been using them throughout the project, so I think its good to stay consistent

Reply:
I just thought they were a bit weird, but I looked into the other files and I see the purpose